spec/requests/api/v3/render_resource_spec.rb

```diff
@@ -83,8 +83,9 @@
             let(:href) { "/work_packages/#{id}" }
             let(:title) { "#{work_package.subject} (#{work_package.status})" }
             let(:text) {
-              "<p>Hello World! Have a look at <a href=\"#{href}\" "\
+              "<p>Hello World! Have a look at <a "\
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.